### DISCUSSION TITLE
Update April 27, 2022 : Diablo II Resurrected Patch 2.4

### DISCUSSION THREAD

**Discussion by fabd** [fabd/diablo2-runewizard#29] [1. UPDATES]
Updated 4 existing runewords, and added 7 new ladder runewords from D2R Patch 2.4.

https://news.blizzard.com/en-us/diablo2/23788293/diablo-ii-resurrected-patch-2-4-now-live#rune

This update adds new weapon types `Daggers` and `Spears` in the runewords table:

- `Spears` weapon type was added to the existing runewords _Obedience_, _Infinity_, _Pride_
- `Daggers` is a weapon type in the new _Plague_ runeword

The `Missile Weapons` weapon type (bows, crossbows) was added to _Insight_.

**Comment by tomasz1986** [comment]
Would it be a problem to mark those changed/added weapon types with the `2.4` tag? At the moment, it's impossible to know which types are the original and which have been newly added.
